### ISSUE TITLE
Layout is more broken in 2.8

### ISSUE THREAD

**Issue by n9** [argoproj/argo-cd#13827]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**

I have just tried [the current master branch](https://github.com/argoproj/argo-cd/tree/50b2f03657026a0987e4910eca4778e8950e6d87). And UI is (more) broken.

**To Reproduce**

Ideally with narrow screen (e.g. 1200 css px).

**Expected behavior**

No errors:) 

**Screenshots**

![image](https://github.com/argoproj/argo-cd/assets/1651867/d3f3d8b8-041f-4aef-b0f4-e572df785e23)

- Two vertical scrollbars on left.
- Clipped top of the status panel on narrow screens (e.g. 1280 css px).
- Minor issue introduced here: https://github.com/argoproj/argo-cd/pull/12068#pullrequestreview-1450594625

**Version**

[the current master branch](https://github.com/argoproj/argo-cd/tree/50b2f03657026a0987e4910eca4778e8950e6d87)

**Logs**

N/A

**Comment by n9** [comment]
Really dirty "solution": https://github.com/n9/argo-cd/commit/cadeb856205019fb15c8d92a1d6c1645eb753386 (and commits after)

(It seems to me that the proper solution might require to refactor [argo-ui](https://github.com/argoproj/argo-ui), which seems to be unmaintained.)

![image](https://github.com/argoproj/argo-cd/assets/1651867/4c549c2f-aaf7-4e9f-89c5-6686d489bf4b)

**Comment by crenshaw-dev** [comment]
argo-ui is... lightly maintained. Would you be up for the refactor work? I have merge permissions on argo-ui.

**Comment by n9** [comment]
I think it will be really complex in order not to break other projects. And as I was adding more tweaks to my "fork", it seems to me it will be better to "refractor" (=> rewrite) the styles of argo-ui and ArgoCD from scratch (rethink its architecture). 

This is not a good task for a random github contributor.

**Comment by crenshaw-dev** [comment]
Makes sense! I'll leave it to @keithchong and @rbreeze to decide how to proceed.

**Comment by n9** [comment]
One more issue that is not directly related to layout (S)CSS, but FE in general:

New UI is laggy. Not sure whether more laggy than in v2.7 (I was using v2.6 before). So it may be already reported here: https://github.com/argoproj/argo-cd/issues/13768

E.g. deleting a pod lags several (10) seconds:

![image](https://github.com/argoproj/argo-cd/assets/1651867/cacae721-8c2d-4ac2-b363-6a79873906f9)

There is something clearly broken there.

**Comment by crenshaw-dev** [comment]
I'm not good at reading front-end debuggers... cab you tell whether the lag is due to front-end problems or slow backend calls?

**Comment by n9** [comment]
Frontend.

Someone is trying to mine bitcoins there :) Even totally broken backend cannot freeze frontend (calls are asynchronous).

**Comment by crenshaw-dev** [comment]
Oh, so it's legit frozen. Might have to do a binary search for the commit, but the biggest front-end change I can think of is the one adding rich YAML editor support.

**Comment by crenshaw-dev** [comment]
Akuity has @rbreeze and Red Hat has @keithchong. @ashutosh16 at Intuit is also pretty handy with front-end work.

**Comment by jsoref** [comment]
fwiw, this appears to have the changes https://github.com/argoproj/argo-cd/compare/release-2.8...n9:argo-cd:patched-2.8.4-1 -- @n9 any plans to make a PR?
